workers/keybroker/broker.go

```diff
@@ -21,68 +17,42 @@ type Closer interface {
 	Close()
 }
 
-// RSAPublicKeyCopier represents behaviour for distributing copies of public keys
-type RSAPublicKeyCopier interface {
-	Copy() rsa.PublicKey
-}
-
 // Config represents broker configuration
 type Config struct {
 	Interval time.Duration
 	Source   Source
 }
 
-var (
-	// DefaultRSA is an empty RSA public key
-	DefaultRSA = &rsa.PublicKey{E: 0, N: big.NewInt(0)}
-)
-
-// NewRSA returns a rsa public key broker based on configuration.
-func NewRSA(config *Config) *RSAPublicKeyBroker {
-	if config == nil {
-		config = &Config{}
-	}
-	if config.Source == nil {
-		config.Source = JWTPublicKeySources
-	}
+func newBroker(keyType string, config *Config) *broker {
```

Comment:
`type KeyType string` so we can have an enum of `Public` and `Private` maybe?
Or is there another reason?